README.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 output: github_document
 ---
-# qCreate
+# qCreate <img src="inst/qCreateLogo.png" align="right" width="220"/>
```

Comment:
I like the logo with the cogs on the side, but maybe improve the three dimensionality by either having a [cutaway revealing the cogs inside](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Fthumbs.dreamstime.com%2Fz%2Fmetal-cog-wheels-gearing-gear-box-167586610.jpg&f=1&nofb=1)?

Reply:
This will be addressed tomorrow by @BBieri, thank you.